features/system-color.yml

```diff
@@ -0,0 +1,10 @@
+name: system-color
```

Comment:
This seems to be a bit more "natural" in use than the hyphenated form. Also, see my commentary on the description:

```suggestion
name: System colors
```